pkg/utils/common.go

```diff
@@ -30,3 +32,17 @@ func GetNamespacedName(obj client.Object) types.NamespacedName {
 func IsDeleted(obj client.Object) bool {
 	return obj != nil && !obj.GetDeletionTimestamp().IsZero()
 }
+
+// GetVersionFromImage return the version from spec.global.image.
+func GetVersionFromImage(image string) string {
+	version := ""
```

Comment:
It seems the `version` isn't necessary. We can do sth like:

```go
if image == "" {
        return ""
}

lastColonIdx := strings.LastIndex(image, ":")
if lastColonIdx < 0 {
        return "latest"
} else {
        return image[lastColonIdx+1:]
}

Reply:
How about this one `fictional.registry.example:10443/imagename`.  I get this example from [k8s document](https://kubernetes.io/docs/concepts/containers/images/#image-names). Following your logic, it will return `10443/imagename`, which isn't correct.

Reply:
For the version, this is one code style that bothers me a lot. Your suggestions look good to me. But I remember some formators will not allow me `return` in `if-else`. So I wonder, in general, we will prefer direct return or return at the end of the function?

Reply:
> Or we don't need to consider this case?

Yes, we do. You're right. In that case, a `latest` should be returned. Let me see... 🤔 We can split the string by `/` and get the tag from the last piece. This should work for all valid image names in a format of 

```
host:port/path/to/repo[:tag]
```

>  in general, we will prefer direct return or return at the end of the function

Yes.

Reply:
done